runtime/vm/jvminit.c

```diff
@@ -2392,7 +2393,11 @@ static UDATA checkArgsConsumed(J9PortLibrary* portLibrary, J9VMInitArgs* j9vm_ar
 		if (IS_CONSUMABLE( j9vm_args, i ) && !IS_CONSUMED( j9vm_args, i )) {
 			char* optString = j9vm_args->actualVMArgs->options[i].optionString;
 			char* envVar = j9vm_args->j9Options[i].fromEnvVar;
-
+			
+			/* If an option is empty or contains only whitespace, it should be ignored instead of rejected */
+			if ((NULL != optString) && isEmpty(optString)) {
```

Comment:
The reference implementation does not ignore empty options if `JavaVMInitArgs.ignoreUnrecognized` is false.

Reply:
@keithc-ca Would it best to just add the `isEmpty()` check to the existing check for ignoreUnrecognized: https://github.com/eclipse/openj9/blob/9453b9844670a117cc67aca2aed5815bec8dc7bf/runtime/vm/jvminit.c#L2401-L2404
i.e.
`if (ignoreUnrecognized && optString && (isEmpty() || !strncmp(optString, "-X", 2) || *optString=='_'))`

Reply:
I have pushed what I mentioned above, matching the reference implementation behaviour for when `JavaVMInitArgs.ignoreUnrecognized` is false.